microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+description: Learn about best practices to use when developing your WebView2 application.
+title: WebView2 development best practices  
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/04/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: WebView2, webview2, WebView, webview, edge, best practices  
+---
+# WebView2 development best practices  
+
+Every development team follows different practices when building their application. When you build WebView2 applications, there are best practices we recommend you follow. This article outlines those recommendations and best practices for you when building production based WebView2 applications.
+
+
+## Use Evergreen WebView2 Runtime  
+
+We generally recommend using the Evergreen WebView2 Runtime instead of the Fixed Version Runtime.  The Evergreen WebView2 Runtime updates automatically and includes the latest features and security patches available to your WebView2 application. The Evergreen WebView2 Runtime also requires less storage space on the disk.
+
+Ensure the Evergreen WebView2 Runtime is installed before using your WebView2 application.  For more information, navigate to [Deploying the Evergreen WebView2 Runtime][Webview2ConceptsDistributionDeployingEvergreenWebview2Runtime].  
+
+## Run regular compatibility tests
+
+When using the Evergreen WebView2 Runtime, ensure you run regular compatibility tests. Because the runtime is automatically updated, you'll want to ensure that your WebView2 application performs as expected.  For more information, navigate to [Stay compatible in Evergreen mode][Webview2ConceptsDistributionStayCompatibleEvergreenMode].  
+
+## Ensure APIs are supported by the installed WebView2 Runtime
+
+WebView2 applications need both a Webview2 SDK, and a WebView2 Runtime installed on the computer to run. Both the SDK and the runtime are versioned. Since APIs are continually being added to WebView2, new versions of the runtime are also released to support the new APIs. You'll need to ensure that the APIs used by your WebView2 application are supported by the WebView2 Runtime that's installed on the computer. 
+
+If you use the Evergreen WebView2 Runtime, consider the connectivity scenarios of your users. Some users may not be continually connected to the internet and may not use an updated runtime. When you push an update to your WebView2 application, the application may break because it uses newer APIs that are not available in the installed runtime. 
+ 
+To solve this situation, you can test for the availability of the APIs in the installed runtime, before making a call to the API. To test for API availability in the installed runtime, use:
```

Comment:
I think we will want code samples in [Determine WebView2 Runtime requirement][Webview2ConceptsVersioningDetermineWebview2RuntimeRequirement] instead of here. + @pagoe-msft.

Reply:
@Reezaali this makes sense to me, to include code snippets in the linked to docs does that seem okay with you?

Reply:
@pagoe-msft - sure! Can you make the update to that doc, and cross-link from this doc please?

Reply:
Ill get some code snippets, lets not block the release of this doc based on this.